OpenRA.Game/Game.cs

```diff
@@ -279,6 +269,25 @@ internal static void Initialize(Arguments args)
 				}
 			}
 
+			GeoIP.Initialize();
+
+			try
+			{
+				if (!Game.Settings.Server.DiscoverNatDevices)
+					Game.Settings.Server.AllowPortForward = false;
+				else
+				{
+					UPnP.DiscoverNatDevices(Settings.Server.NatDiscoveryTimeout).Wait();
```

Comment:
`DiscoverNatDevices` returns a `Task`, so why can't you assign that to a variable and then `Wait()` on it at the end of the loading?